Libplanet.Tests/Net/SwarmTest.cs

```diff
@@ -769,42 +1042,37 @@ public async Task CanBroadcastBlock()
 
             try
             {
-                await StartAsync(swarmA);
-                await StartAsync(swarmB);
-                await StartAsync(swarmC);
+                await RunAsync(swarmA);
+                await RunAsync(swarmB);
+                await RunAsync(swarmC);
 
-                await swarmA.AddPeersAsync(new[] { swarmB.AsPeer });
-                await swarmA.AddPeersAsync(new[] { swarmC.AsPeer });
+                await BootstrapAsync(swarmB, swarmA.AsPeer);
+                await BootstrapAsync(swarmC, swarmA.AsPeer);
 
-                await EnsureExchange(swarmA, swarmB);
-                await EnsureExchange(swarmA, swarmC);
-                await EnsureExchange(swarmB, swarmC);
+                Log.Debug(swarmA.TraceTable());
+                Log.Debug(swarmB.TraceTable());
+                Log.Debug(swarmC.TraceTable());
 
                 swarmB.BroadcastBlocks(new[] { chainB.Last() });
 
-                await swarmC.BlockReceived.WaitAsync();
-                await swarmA.BlockReceived.WaitAsync();
-
-                Assert.Equal(chainB.AsEnumerable(), chainC);
```

Comment:
is this intended?

Reply:
Yes, because swarmB may be only connected to swarmA, than swarmC's `BlockReceived` will never be invoked.